script.js

```diff
@@ -0,0 +1,7 @@
+const parse = require('gemini-to-html/parse')
+const render = require('gemini-to-html/render')
+if (location.href.startsWith('gemini')) {
```

Comment:
Not all things served over gemini are gemini pages, maybe we could do the same content-type detection as in the markdown renderer?

Reply:
Oh okay, should be easy. Will we assume that gemini pages can be served over other protocols?

Reply:
Resolved assuming Gemini is server over all protocols